### PR TITLE
Raised PR to merge CI Images 3.0 to debug/paas

### PR DESCRIPTION
**Pull request by priyadishah**

### CODE REVIEW

**Review by robe070 — CHANGES_REQUESTED** [review]
empty

File: Templates/releaseTemplate.yml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            displayName: Checkout nkhl-accolite/cookbooks
+
+          - task: PowerShell@2
+            displayName: 'Artifact Check : Set Gate Variable'
+            inputs:
+              targetType: filePath
+              filePath: './$(System.DefaultWorkingDirectory)/_robe070_cookbooks/scripts/azure_set_gate_variable_cloud_account_id.ps1'
+              arguments: '-Version "$(System.JobDisplayName)" -osName "Windows Server 2019"'
+              
+          #Your build pipeline references an undefined variable named ‘Gate.IsEnabled’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab. See https://go.microsoft.com/fwlink/?linkid=865972
```
Comment (robe070) [inline Templates/releaseTemplate.yml]: There is some messy code in here...

Needs fixing.

File: scripts/bake-ide-ami.ps1 [hunk 2]
```diff
@@ -679,7 +679,7 @@ $jsonObject = @"
 
 
         if ( $InstallBaseSoftware ) {
-
+
```
Comment (robe070) [inline scripts/bake-ide-ami.ps1]: bake-ide-ami.ps1 changes can be deleted.

File: scripts/getchoco.ps1 [hunk 3]
```diff
@@ -33,6 +33,7 @@ $url = ''
 
 try {
     Write-Host("$(Log-Date) On Azure, choco 2.0 requires .Net 4.8. Firstly this requires the caller to action a reboot. Then when executing choco from a remote session it fails to locate .Net 4.8. If you RDP in, choco works fine. Note that AWS does not have this issue so there is a setting somewhere but this could not be located.")
```
Comment (robe070) [inline scripts/getchoco.ps1]: These changes can be deleted too.

File: Templates/releaseTemplate.yml [hunk 4]
```diff
@@ -0,0 +1,186 @@
+parameters:
```
Comment (robe070) [inline Templates/releaseTemplate.yml]: This file is in the wrong directory

**Review by robe070 — CHANGES_REQUESTED** [review]
Fix up the comments I've left.

Run it again.

File: Templates/releaseTemplate.yml (hunk 4 above)
Comment (priyadishah) [reply Templates/releaseTemplate.yml]: removed this file from the directory.

File: scripts/getchoco.ps1 (hunk 3 above)
Comment (priyadishah) [reply scripts/getchoco.ps1]: removed changes.

File: Templates/releaseTemplate.yml (hunk 1 above)
Comment (priyadishah) [reply Templates/releaseTemplate.yml]: removed this file from the directory.

**Comment by priyadishah** [comment]
> empty

Nikhil might be created file but no file there.

File: scripts/bake-ide-ami.ps1 (hunk 2 above)
Comment (priyadishah) [reply scripts/bake-ide-ami.ps1]: Hi @robe070 removed changes.

**Comment by priyadishah** [comment]
> Fix up the comments I've left.
> 
> Run it again.

Hi @robe070,
cc @nkhl-accolite 

after all changes, I ran pipeline again and it's working.

https://dev.azure.com/VisualLansa/Lansa%20Azure%20Scalable%20License%20Images/_build/results?buildId=2988&view=results

File: scripts/getchoco.ps1 [hunk 5]
```diff
@@ -32,7 +32,7 @@
 $url = ''
 
 try {
-    Write-Host("$(Log-Date) On Azure, choco 2.0 requires .Net 4.8. Firstly this requires the caller to action a reboot. Then when executing choco from a remote session it fails to locate .Net 4.8. If you RDP in, choco works fine. Note that AWS does not have this issue so there is a setting somewhere but this could not be located.")
```
Comment (robe070) [inline scripts/getchoco.ps1]: Revert this. Its a comment thats clearly seen in the log file as to why choco 2.0 is not being used.

**Review by robe070 — CHANGES_REQUESTED** [review]
None of the bake-ide-ami.ps1 nor getchoco.ps1 changes are needed. Its all just spacing or plain wrong comments - i.e. .Net install comment is not correct..

File: scripts/getchoco.ps1 (hunk 5 above)
Comment (priyadishah) [reply scripts/getchoco.ps1]: reverted change and confirmed with Tarun.

**Comment by priyadishah** [comment]
> None of the bake-ide-ami.ps1 nor getchoco.ps1 changes are needed. Its all just spacing or plain wrong comments - i.e. .Net install comment is not correct..

Hi @robe070 removed empty spaces from bakde-ide-ami.ps1 and confirmed with Tarun.

**Review by robe070 — CHANGES_REQUESTED** [review]
scripts/bake-ide-ami.ps1 changes are NOT required. Remove them. The file should not be listed. It should be unchanged.

**Comment by priyadishah** [comment]
> scripts/bake-ide-ami.ps1 changes are NOT required. Remove them. The file should not be listed. It should be unchanged.

Hi @robe070 changes done.

**Pull request merged**